.goreleaser.yml

```diff
@@ -86,6 +84,7 @@ nfpms:
       ./caddy-dist/welcome/index.html: /usr/share/caddy/index.html
     config_files:
       ./caddy-dist/config/Caddyfile: /etc/caddy/Caddyfile
+      ./caddy-dist/scripts/completions/bash-completion: /etc/bash_completion.d/caddy
```

Comment:
Actually this should be a `files`, not `config_files`. The difference is that `apt remove` doesn't remove `config_files`, but does remove `files`, and `apt purge` removes both. Doesn't make sense to keep bash completions around after remove.